sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonScanner.java

```diff
@@ -69,6 +76,7 @@ public class PythonScanner extends Scanner {
   private final PythonCpdAnalyzer cpdAnalyzer;
   private final PythonIndexer indexer;
 
+  private InputFile inputFile;
```

Comment:
The rest of `PythonScanner` does not rely on such a state. I would prefer to remove this field.